docs/modules/ROOT/pages/usage/lsp.adoc

```diff
@@ -0,0 +1,47 @@
+= LSP (Language Server Protocol)
+
+NOTE: The built-in language server was introduced in RuboCop 1.53.
```

Comment:
You may want to mention that we'll consider this experimental feature for a while.

Reply:
Sure! I updated it.